recipes/fastjet-contrib/meta.yaml

```diff
@@ -0,0 +1,132 @@
+{% set name = "fastjet-contrib" %}
+{% set version = "1.056" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://fastjet.hepforge.org/contrib/downloads/fjcontrib-{{ version }}.tar.gz
+  sha256: fc31544424dece0d0676ea2433ad1e96fd9db82920bc7a7ef6294ce94e659d6e
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}
+
+    build:
+      skip: true  # [win]
+      run_exports:
+        - {{ pin_subpackage('fastjet-contrib', max_pin='x.x') }}
+      script:
+        # Get an updated config.sub and config.guess
+        - cp $BUILD_PREFIX/share/gnuconfig/config.* .
+        - ./configure --help
+        - ./configure --list
+        - ./configure --fastjet-config=$(command -v fastjet-config) --prefix=$PREFIX CXX=$CXX
+        - make --jobs="${CPU_COUNT}"
+        - make fragile-shared --jobs="${CPU_COUNT}"
+        - make check --jobs="${CPU_COUNT}"  # [build_platform == target_platform]
+        - make install
+        - make fragile-shared-install
+        - make clean
+        # Remove the static libraries that got installed
+        - rm $PREFIX/lib/libCentauro.a
+        - rm $PREFIX/lib/libClusteringVetoPlugin.a
+        - rm $PREFIX/lib/libConstituentSubtractor.a
+        - rm $PREFIX/lib/libEnergyCorrelator.a
+        - rm $PREFIX/lib/libFlavorCone.a
+        - rm $PREFIX/lib/libGenericSubtractor.a
+        - rm $PREFIX/lib/libJetCleanser.a
+        - rm $PREFIX/lib/libJetFFMoments.a
+        - rm $PREFIX/lib/libJetsWithoutJets.a
+        - rm $PREFIX/lib/libLundPlane.a
+        - rm $PREFIX/lib/libNsubjettiness.a
+        - rm $PREFIX/lib/libQCDAwarePlugin.a
+        - rm $PREFIX/lib/libRecursiveTools.a
+        - rm $PREFIX/lib/libScJet.a
+        - rm $PREFIX/lib/libSignalFreeBackgroundEstimator.a
+        - rm $PREFIX/lib/libSoftKiller.a
+        - rm $PREFIX/lib/libSubjetCounting.a
+        - rm $PREFIX/lib/libValenciaPlugin.a
+        - rm $PREFIX/lib/libVariableR.a
+
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - automake
+        - libtool
+        - make
+        - pkg-config
+        - gnuconfig
+        - fastjet-cxx  # [build_platform != target_platform]
+      host:
+        - fastjet-cxx
+      run:
+        - fastjet-cxx
+
+    test:
+      requires:
+        - {{ compiler('cxx') }}
+        - curl
+        - tar
+        - grep
+        - sed
+      script: run_test_shared.sh
```

Comment:
These tests never get run, and I don't understand why.

Reply:
@conda-forge/staged-recipes can you offer any advice? I've read https://conda-forge.org/docs/maintainer/knowledge_base/#multi-output-recipes and looked at https://github.com/conda-forge/libarchive-feedstock/blob/main/recipe/meta.yaml when doing local debugging, but I'm still confused as to why only one of the two package tests get run.